be/src/vec/functions/array/function_array_aggregation.cpp

```diff
@@ -146,6 +146,18 @@ struct ArrayAggregateImpl {
         using Function = AggregateFunction<AggregateFunctionImpl<operation>>;
         const DataTypeArray* data_type_array =
                 static_cast<const DataTypeArray*>(remove_nullable(arguments[0]).get());
+        if constexpr (operation == AggregateOperation::AVERAGE ||
```

Comment:
I think you shoud use witelist to check instead of blacklist, since if someone add a new agg operation it passed by default.